### ISSUE TITLE
ida-free 8.1 install failed

### ISSUE THREAD

**Issue by zhangchaoza** [ScoopInstaller/Extras#9421]
## Bug Report

**Package Name:** ida-free

### Current Behaviour

```
$ sudo scoop install ida-free
```

Installation aborted

![屏幕截图 2022-10-08 143106](https://user-images.githubusercontent.com/8922723/194693396-a81afce5-4254-42bd-bc10-a13bb5e4f45c.png)

### Expected Behaviour

install success

### Additional context/output

```
Installing 'ida-free' (8.1) [64bit] from extras bucket
Loading idafree81_windows.exe from cache
Checking hash of idafree81_windows.exe ... ok.
Running pre_install script...
Running installer... error.
ERROR Exit code was 1!
Installation aborted. You might need to run 'scoop uninstall ida-free' before trying again.
```

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:**7.2.6

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# Your configuration here
```

**Comment by Zliced13** [comment]
Fixed in #9436

**Comment by Zliced13** [comment]
Hey you can try this fixed version in my Personal bucket if you want to:
```
scoop bucket add MyScoop 'https://github.com/Zliced13/MyScoop'
scoop install ida-free_fixed
```
